MessengerProj/src/main/res/values/strings.xml

```diff
@@ -417,14 +417,14 @@
     <string name="Never">Never</string>
     <string name="AutocryptKeyTransferInitiate">Initiate Autocrypt Key Transfer</string>
     <string name="AutocryptKeyTransfer">Autocrypt Key Transfer</string>
-    <string name="AutocryptKeyTransferMsgBeforeSend">The <![CDATA[<i>]]>Autocrypt Key Transfer<![CDATA[</i>]]> requires that the mail client on the other device is Autocrypt-compliant.\n\nYou can then send your secret key to yourself. The key will be encrypted by a setup code which is displayed here and must be typed on the other device.</string>
-    <string name="AutocryptKeyTransferMsgAfterSend">Your key has been sent to yourself. Switch to the other device and open the setup message. You should be prompted for a setup code. Type the following digits into the prompt:\n\n%1$s\n\nOnce you\'re done, your other device will be ready to use Autocrypt.</string>
+    <string name="AutocryptKeyTransferMsgBeforeSend">The <![CDATA[<i>]]>Autocrypt Key Transfer<![CDATA[</i>]]> requires the e-mail client on the other device to be Autocrypt-compliant.\n\nYou can then send your secret key to yourself. It will be encrypted by a setup code, displayed here to be typed onto the other device.</string>
+    <string name="AutocryptKeyTransferMsgAfterSend">Your key has been sent to yourself. Open the setup message on the other device and type the following digits:\n\n%1$s\n\nOnce done, your other device will be ready to use Autocrypt.</string>
     <string name="AutocryptSetupMessageSubject">Autocrypt Setup Message</string>
-    <string name="AutocryptSetupMessageGeneralBody">This is the Autocrypt Setup Message used to transfer your key between clients.\n\nTo decrypt and use your key, open the message in an Autocrypt-compliant client and enter the setup code presented on the generating device.</string>
-    <string name="AutocryptSetupMessageTapBody">This is the Autocrypt Setup Message used to transfer your key between clients.\n\nTo decrypt and use your key, tap on this message.</string>
-    <string name="AutocryptKeyTransferPleaseEnterCode">Please enter the setup code that is displayed on the other device.</string>
-    <string name="AutocryptKeyTransferBadCode">Bad setup code. Please try again.\n\nIf you do not remember the setup code, just initiate the key transfer on the other device again.</string>
-    <string name="AutocryptKeyTransferSucceeded">Key transferred successfully. This device is now ready to use Autocrypt with the same key as the other device.</string>
+    <string name="AutocryptSetupMessageGeneralBody">The setup code must be entered on the other device to use your key there.</string>
```

Comment:
I think, this is a little too short. The string will be the body of a message that may lack all context  of the app.
The hint with the Autocrypt-compliant client is important. Eg. the user may see the message in Thunderbird without Enigmail where taping in the message does not work.

Reply:
AutocryptSetupMessage_Subject_ states it is in fact the Autocrypt setup message
With this being AutocryptSetupMessageGeneral_Body_
See other comment. I am unsure how or what to reinstate, because why does the tapping appear out of context?

Reply:
The tapping may just not be possible if the client is not autocrypt-capable. But the client will display the text nevertheless in this case. So the text should give the user a hint into the right direction. In fact, when displaying the message with an autocrypt-capable client, this client may replace the text by its own - eg. Thunderbird without Enigmail displays our text, Thunderbird with Enigmail its own text.